lib/api_client/connection/middlewares/request/logger.rb

```diff
@@ -46,7 +46,19 @@ def gather_response_debug_lines(response, taken_sec, debug_lines)
     debug_lines
   end
 
-  def current_stamp_millisec
-    Process.clock_gettime(Process::CLOCK_MONOTONIC, :millisecond)
+  class CurrentTimestamp
+    class << self
+      version = RUBY_VERSION.split('.').map(&:to_i)
+
+      if (version[0] < 2) || (version[0] == 2 && version[1] < 2)
+        def milis
+          Time.now.to_f * 1000
```

Comment:
This is going to return a `Float`, while the `#clock_gettime` returns `Fixnum`. I would do `to_i` on this, just to be safe.